actionview/lib/action_view/helpers/sanitize_helper.rb

```diff
@@ -166,30 +165,6 @@ def link_sanitizer
         def white_list_sanitizer
           @white_list_sanitizer ||= sanitizer_vendor.white_list_sanitizer.new
         end
-
-        ##
-        # :method: sanitized_allowed_tags=
```

Comment:
They were "documented" here even though the implementation has moved
elsewhere, but they were documented down inside the no-doc'd ClassMethods.
So I moved the docs for doing this up into the main documentation above so
that they would be publicly accessible.  (They were also previously
documented above as well, albeit somewhat inaccurately in the original.)

I can revert if necessary. Just let me know!

On Friday, January 16, 2015, Zachary Scott <notifications@github.com> wrote:

> In actionview/lib/action_view/helpers/sanitize_helper.rb
> <https://github.com/rails/rails/pull/18557#discussion-diff-23119236>:
>
> > @@ -166,30 +165,6 @@ def link_sanitizer
> >          def white_list_sanitizer
> >            @white_list_sanitizer ||= sanitizer_vendor.white_list_sanitizer.new
> >          end
> > -
> > -        ##
> > -        # :method: sanitized_allowed_tags=
>
> Where did these go?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/18557/files#r23119236>.
>